index.js

```diff
@@ -40,7 +40,8 @@ function DB (opts) {
     map: function (row, next) {
       if (!row.value) return null
       var v = row.value.v, d = row.value.d
-      if (v && v.lat !== undefined && v.lon !== undefined) {
+      var k = row.value.k
+      if (k && v.lat !== undefined && v.lon !== undefined) {
```

Comment:
Now that DELETEs can have a `v` (value) as well as PUTs, checking for `k` (key) is a more reliable way of differentiating between PUTs and DELETEs. I can wrap this in a helper function to be more explicit -- not a bad idea.